homeassistant/components/kodi/config_flow.py

```diff
@@ -105,7 +105,7 @@ async def async_step_zeroconf(
     ) -> FlowResult:
         """Handle zeroconf discovery."""
         self._host = discovery_info.host
-        self._port = int(discovery_info.port)
+        self._port = discovery_info.port
```

Comment:
self._port allows None, and `discovery_info.port` is `int | None`, so this fits.
I wonder if it wouldn't be better to adjust this to
```python
self._port = discovery_info.port or DEFAULT_PORT
```

Reply:
Yes, fallback to the default is better.

Reply:
I've added the fallback.